lib/record_store/cli.rb

```diff
@@ -210,7 +210,7 @@ def validate_records
         end
       end
 
-      if invalid_zones.size > 0
+      if !invalid_zones.size.empty?
```

Comment:
```suggestion
      unless invalid_zones.size.empty?
```

Or even better, so it's nicer to read, we can revert the condition:

```ruby
      if invalid_zones.size.empty?
        puts 'All zones have valid definitions.'
      else
        abort "The following zones were invalid: #{invalid_zones.join(', ')}"
      end
```